Header header
string robot_id
bool is_ready
float64 x
float64 y